src/serde_tests.rs

```diff
@@ -848,34 +848,64 @@ fn dictionary_serialize_xml() {
     let buf = buf.into_inner();
     let xml = std::str::from_utf8(&buf).unwrap();
 
-    let comparison = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>
-<!DOCTYPE plist PUBLIC \"-//Apple//DTD PLIST 1.0//EN\" \"http://www.apple.com/DTDs/PropertyList-1.0.dtd\">
-<plist version=\"1.0\">
+    let comparison = r#"<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I've had problems in the past with text editors replacing tabs with spaces. I'd rather keep the escaped version, even though it doesn't look very nice.

Reply:
converted it back, seems reasonable to me